src/parser/mod.rs

```diff
@@ -10983,17 +11069,23 @@ impl<'a> Parser<'a> {
         })
     }
 
-    /// Parse an expression, optionally followed by ASC or DESC (used in ORDER BY)
-    pub fn parse_order_by_expr(&mut self) -> Result<OrderByExpr, ParserError> {
-        let expr = self.parse_expr()?;
-
-        let asc = if self.parse_keyword(Keyword::ASC) {
+    /// Parsae ASC or DESC, returns an Option with true if ASC, false of DESC or `None` if none of
+    /// them.
+    pub fn parse_asc(&mut self) -> Option<bool> {
```

Comment:
Since it was used to only represent if asc (true/false) I wanted to use something like `is_asc` but I found zero examples of `is_` prefixed methods so stuck to `parse`. I think `parse_asc_desc` is fine but it's a bit confusing that it doesn't return an enum or either of them but a boolean. Hopefully the description is good enough, I opted to use `parse_asc_desc`!